addons/website_event_sale/models/sale_order.py

```diff
@@ -92,9 +92,10 @@ def _cart_update(self, product_id=None, line_id=None, add_qty=0, set_qty=0, **kw
         # adding attendees
         elif ticket and new_qty > old_qty:
             line = OrderLine.browse(values['line_id'])
-            line._update_registrations(confirm=False, cancel_to_draft=True, registration_data=kwargs.get('registration_data', []))
-            # add in return values the registrations, to display them on website (or not)
-            values['attendee_ids'] = self.env['event.registration'].search([('sale_order_line_id', '=', line.id), ('state', '!=', 'cancel')]).ids
+            if registration_update:
+                line._update_registrations(confirm=False, cancel_to_draft=True, registration_data=kwargs.get('registration_data', []))
```

Comment:
instead of adding registration_update param in the method, we could use registration_data from kwargs.
```
if kwargs.get('registration_data'): 
    line._update_registrations(confirm=False, cancel_to_draft=True, registration_data=kwargs['registration_data'])
    # add in return values the registrations, to display them on website (or not)
    values['attendee_ids'] = self.env['event.registration'].search([('sale_order_line_id', '=', line.id), ('state', '!=', 'cancel')]).ids
```

Reply:
done